unblob/models.py

```diff
@@ -147,6 +148,27 @@ def as_report(self, entropy: Optional[EntropyReport]) -> UnknownChunkReport:
         )
 
 
+@attr.define(repr=False)
+class PaddingChunk(Chunk):
+    r"""Gaps between valid chunks or otherwise unknown chunks.
+
+    Important for manual analysis, and analytical certanity: for example
+    entropy, other chunks inside it, metadata, etc.
+
+    These are not extracted, just logged for information purposes and further analysis,
+    like most common bytes (like \x00 and \xFF), ASCII strings, high entropy, etc.
+    """
+
+    def as_report(self, entropy: Optional[EntropyReport]) -> PaddingChunkReport:
+        return PaddingChunkReport(
```

Comment:
Maybe we could reuse `ChunkReport` with `handler_name` `padding`?

Reply:
Smart !

Reply:
Implemented this change and adapted both `test_chunk_in_chunk_report_structure` and `test_flat_report_structure`.